src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -4,14 +4,30 @@
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Intake extends SubsystemBase {
   /** Creates a new Intake. */
-  public Intake() {}
+
+   CANSparkMax motorController1; 
+   CANSparkMax motorController2; 
+
+  public Intake() {
+    motorController1 = new CANSparkMax(0, MotorType.kBrushless);
+    motorController2 = new CANSparkMax(1, MotorType.kBrushless);
+  }
+
+  public void intakeBalls(double speed){
+    motorController1.set(speed);
+    motorController2.set(-speed);
```

Comment:
Because of how the intake is going to work, you'll need to be able to control the direction of each individual motor, so it should probably be two separate methods, one for each motor

You'll also need two different commands, one for intaking the cargo which you technically have already, but also another command to feed the ball to the shooter. Ask Nathan about which motor spins which direction for each command. 

Also create a stop method for the intake